src/main/java/oop/animals/mammals/predators/Predator.java

```diff
@@ -0,0 +1,10 @@
+package oop.animals.mammals.predators;
+
+import oop.animals.mammals.Mammal;
+
+public abstract class Predator extends Mammal {
+
+    enum WeaponType {
```

Comment:
From the task: each predator should have property primaryWeaponType, which can be claws or teeth. This enum only stores possible values.